src/app/services/user.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@angular/core";
+import { Devices } from "app/models/devices";
+import { SpotifyService } from "app/services/spotify.service";
+import { User } from "app/models/user";
+
+@Injectable()
+export class UserService {
+  devices: Devices = new Devices();
+  user: User = new User();
+
+  constructor(private spotifyService: SpotifyService) {
+    this.getUser();
+    this.getUserDevices();
+  }
+
+  public getUser() {
```

Comment:
I think it applies the same comment on the get methods...we can change it to  "load" maybe... also.. is this method used outside the class?  If it is called only in the init method,  can we change them to "private"? (I am not sure, maybe we can verify with Jorge).